frontend/src/components/HelpPopup/HelpPopup.tsx

```diff
@@ -0,0 +1,140 @@
+        }
+      }
+    }
+  }
+
+  /**
+   * gets the list of renders
+   */
+  function getRenders() {
+    const listOfTitles = ["File Upload", "Processed Files", "Predicted Files"];
+    const listOfDescriptions = [
+      "Step 1: Drag and drop your files into the upload box. Once uploaded your files will appear in the Uploaded Files section on the right. You can select the file you want to process and click the process button. Once you have processed your data it will be saved. You do not need to process it again. Click Go To Processed Files to move to the processed data menu.",
+      "Step 2 - Data prediction and download: In this step you can either download the.csvof your Apple Watch or Fitbit data or, use our machine learning methods to predict lying, sitting, and walking at difference intensities. Select the file you want to predict, select the machine learning model for the prediction (we recommend Random Forrest) and click the Predict File button. Once prediction is complete, move to the predicted files page.",
+      "Step 3 - Predicted data files: On this page, you can download your new data files with both the raw Apple Watch or Fitbit data, the features we use for our machine learning models, and the predicted activity for each minute of your data. If you want to upload different files, go back to the file upload page. You can also click on individual steps to rerun a different machine learning model if you want.",
+    ];
+
+    let j = -1;
+    renders = listOfTitles.map((i) => {
+      j += 1;
+
+      return (
+        <div className={styles.textGroup}>
+          <div className={styles.dropDownTitle}>
+            <text>{i}</text>
+            <button
+              className={styles.dropDownButton}
+              type="button"
+              onClick={toggleTextDisplay}
+              value={j}
+              id={`button ${j}`}
+            >
+              v
+            </button>
+          </div>
+          <text
+            className={styles.paragraph}
+            id={`Paragraph ${j}`}
+            values="closed"
+          >
+            {listOfDescriptions[j]}
+          </text>
+        </div>
+      );
+    });
+  }
+
+  function openPopup() {
+    const popup = document.getElementById("popup");
+
+    // console.assert(popup != null);
+    if (popup == null) {
+      rollbar.error("Help popup element does not exist in document");
+    } else {
+      popup.style.display = "block";
+    }
+  }
+
+  function closePopup() {
+    const popup = document.getElementById("popup");
+
+    // console.assert(popup != null);
+    if (popup == null) {
+      rollbar.error("Help popup element does not exist in document");
+    } else {
+      popup.style.display = "none";
+    }
+  }
+
+  getRenders();
+
+  return (
+    <div>
+      <button
+        className={styles.helpButton}
+        type="button"
+        onClick={openPopup}
+        id="popupButton"
```

Comment:
Also add a data-testid attribute here. This will make it easier for our jest test to find the element.

We can do something like data-testid="popupButton"

So if we do something like getByTestId("popupButton") Its easier to find this element